frontend/src/Settings/Settings.css

```diff
@@ -0,0 +1,9 @@
+.form-label {
```

Comment:
I'm not sure about it, but may be it will be better to use one .css file for different pages instead of typing the same styles for different pages...

Reply:
I changed the login component where almost this css was also used, so there is no double-use now.

Here I couldn't quickly achieve the display that I want without this CSS, so I am keeping the CSS for this component.

(display that I want is that all labels of the settings align to the left and all settings input fields align tot he right)